Fix redirectUri with decodeURIComponent

m0067 opened the pull request:
Problem: retrieving a URL from a query parameter contains encoded characters (%25, %2F). When you pass redirectUri to window.open, the browser interprets the percent signs literally as part of the URL instead of decoding them back to their original characters.
Solution: use decodeURIComponent()

paulomarg reviewed (commented):
Thanks for contributing, just one nit!

m0067 commented on pages/ExitIframe.jsx:
```diff
@@ -20,7 +20,7 @@ export default function ExitIframe() {
         url.hostname.endsWith(".myshopify.com")
       ) {
-        window.open(redirectUri, "_top");
+        window.open(decodeURIComponent(redirectUri), "_top");
```
Sure. Done.

m0067 commented:
> Thanks for contributing, just one nit!
> 
> Also, you'll need to sign the CLA so we can merge this :)

I updated my first commit to follow CLA.
I added a second commit as per the comment about using url instead of decodeURIComponent(redirectUri).
